example.setup.config.json

```diff
@@ -0,0 +1,18 @@
+{
+  "applicationName": "Awesome App",
```

Comment:
Let's keep everything empty string, including the `accessKey` and `secretAccessKey` properties. This way we can:

1. Throw an error if `applicationName` isn't defined
1. Throw an error if no profile entries were filled in